server/controllers/groups.js

```diff
@@ -311,10 +311,13 @@ export const create = (req, res, next) => {
     .tap(g => {
       // if there is a host id provided, we add the collective to the host
       if (group.HostId) {
-        return User.findOne({ where: { id: group.HostId }}).tap(host => _addUserToGroup(g, host, {role: roles.HOST, remoteUser: creator}))
+        return User.findOne({ where: { id: group.HostId }}).tap(h => {
+          host = h;
```

Comment:
We pass it to the activity data. I just reused the same pattern used in that segment for groupCreated. I think it's clearer and easier to read/follow.